src/components/input/help-text.jsx

```diff
@@ -1,19 +1,38 @@
 import React, { PropTypes } from 'react';
 import classNames from 'classnames';
-// TODO: Move SCSS into JSS
-// import './help-text.scss';
+import { createStyleSheet } from 'jss-theme-reactor';
 
-function HelpText(props) {
+const styleSheet = createStyleSheet('HelpText', (theme) => {
+  const { palette } = theme;
+
+  return {
+    'help-text': {
+      position: 'absolute',
+      display: 'block',
+      width: '100%',
+      textAlign: 'right',
+      bottom: '-16px',
+      fontSize: '12px',
+      color: palette.text.muted,
+      '&.help-text--has-success': { color: palette.variant.success },
```

Comment:
`'&.help-text--has-success': { color: palette.variant.success },` -> `success: { color: palette.variant.success }` (outside of 'help-text')